ext/oboe_metal/src/frames.h

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2021 SolarWinds, LLC.
+// All rights reserved.
+
+#ifndef FRAMES_H
+#define FRAMES_H
+
+#include <vector>
+
+#include <mutex>
+#include <unordered_map>
+
+#include <ruby/ruby.h>
+#include <ruby/debug.h>
+
+#include "profiling.h"
+#include "oboe_api.hpp"
+
+using namespace std;
```

Comment:
From my experiences, it's not a good idea to place `using namespace std;` in a header file. I understand that it's convenient but it can have strange side effects that are hard to trouble shoot. This is because everywhere this header gets included will also be using this namespace. In some cases, there will will name conflicts with methods that are found in `std` but are defined in other places as well. But it's up to you how you want to handle it.

Reply:
thanks - that's good advice, will change